package.json

```diff
@@ -19,9 +19,9 @@
     "compile:main": "webpack --config webpack.main.ts",
     "compile:renderer": "webpack --config webpack.renderer.ts",
     "compile:i18n": "lingui compile",
-    "build:linux": "yarn compile && electron-builder --linux --dir -c.productName=LensDev",
-    "build:mac": "yarn compile && electron-builder --mac --dir -c.productName=LensDev",
-    "build:win": "yarn compile && electron-builder --win --dir -c.productName=LensDev",
+    "build:linux": "yarn compile && electron-builder --linux --dir -c.productName=Lens",
```

Comment:
What was the benefit of using `LensDev` as `productName`?

Reply:
I think it originates from electron-webpack (not used anymore) where we did have `Lens` for production and `Electron` as dev/test.